scripts/src/java/org/oppia/android/scripts/SyntaxErrorHandler.kt

```diff
@@ -0,0 +1,41 @@
+package org.oppia.android.scripts
+
+import org.xml.sax.ErrorHandler
+import org.xml.sax.SAXParseException
+
+/**
+ * Class for custom error handling of the parse exception thrown
+ * by the parser, to log the failure message in a cleaner way.
+ */
+class SyntaxErrorHandler(val repoPath: String) : ErrorHandler {
```

Comment:
Does ``repoPath`` need to be public? In general, prefer to keep everything with as minimal visibility as necessary to simplify your public contract (put another way: everything that's public needs to be tested thoroughly since other components can use them).

Reply:
Also, please add tests for this file.

Reply:
Outdated. Tests added too.